roles/georchestra/templates/mviewerstudio/mviewerstudio.service.j2

```diff
@@ -0,0 +1,22 @@
+[Unit]
+ Description=mviewerstudio
+ After=network.target
+
+ [Service]
+ User=mviewer
+ Environment="EXPORT_CONF_FOLDER=/srv/data/mviewer/apps/store/"
+ Environment="CONF_PUBLISH_PATH_FROM_MVIEWER=apps/prod"
+ Environment="CONF_PATH_FROM_MVIEWER=apps/store"
+ Environment="MVIEWERSTUDIO_PUBLISH_PATH=/srv/data/mviewer/apps/prod"
+ Environment="DEFAULT_ORG=public"
+ Environment="LOG_LEVEL=INFO"
+ WorkingDirectory=/srv/apps/mviewerstudio/srv/python
+ ExecStart=/srv/apps/mviewerstudio/srv/python/.venv/bin/gunicorn \
+     -b 127.0.0.1:{{ mviewerstudio.port}} \
+     --access-logfile /{{ logs_basedir }}/mviewerstudio-access.log \
```

Comment:
extra leading slash.. also, to avoid having to chmod 775 `logs_basedir`, itd be better to use `{{ logs_basedir }}/mviewerstudio/access.log` this way only ``{{ logs_basedir }}/mviewerstudio/` needs to be writable by the mviewer user.

same thing for `--error-logfile` below